### PR TITLE
Replace status strings to status arrays in a getAgentChannels method.

### PR DESCRIPTION
**Pull request by dzeckelev**
Resolves BV-1133.

### CODE REVIEW

File: ui/channel.go [hunk 1]
```diff
@@ -187,8 +187,17 @@ func (h *Handler) GetClientChannels(password, channelStatus,
 	if err := h.checkPassword(logger, password); err != nil {
 		return nil, err
 	}
+	// TODO(maxim): Replace channelStatus, serviceStatus arguments types from string to []string.
```
[the remark this answers is not in the document]
Comment (furkhat) [reply ui/channel.go]: resolved?

File: ui/channel.go [hunk 2]
```diff
@@ -342,30 +351,44 @@ func (h *Handler) createClientChannelResult(logger log.Logger,
...
+		condition := statusCondition(channelStatuses, "channel_status")
+		conditions = append(conditions, condition)
+		for _, channelStatus := range channelStatuses {
+			args = append(args, channelStatus)
+		}
 	}
 
-	items := h.tailElements(conditions)
+	if len(serviceStatuses) != 0 {
+		condition := statusCondition(serviceStatuses, "service_status")
```
Comment (furkhat) [inline ui/channel.go]: a little duplication. Can we extract whole `if` with its body to a method, and just call the method twice? Smth like this:
```
args = append(method(channelStatuses, "channel_status"), args...)
args = append(method(serviceStatuses, "service_status"), args...)
```

File: ui/channel.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (dzeckelev) [reply ui/channel.go]: Now it is not necessary.

**Comment by dzeckelev** [comment]
@furkhat updated.

File: doc/ui/rpc.md [hunk 3]
```diff
@@ -354,8 +354,8 @@ curl -X POST -H "Content-Type: application/json" --data '{"method": "ui_changeCh
 
 *Parameters*: 
 1. Password (string)
-2. Channel status (string)
-3. Service status (string)
+2. Channel statuses (array of `string`s or empty array)
```
Comment (ababo) [inline doc/ui/rpc.md]: I would write "array of `string`s, can be empty".

File: ui/channel.go [hunk 4]
```diff
@@ -187,8 +187,17 @@ func (h *Handler) GetClientChannels(password, channelStatus,
 	if err := h.checkPassword(logger, password); err != nil {
 		return nil, err
 	}
+	// TODO(maxim): Replace channelStatus, serviceStatus arguments types from string to []string.
+	chStatuses := make([]string, 0)
```
Comment (ababo) [inline ui/channel.go]: `var chStatuses []string` of `chStatuses:=[]string{}` is shorter, up to you.

**Pull request merged**